perfkitbenchmarker/benchmarks/cassandra_ycsb_benchmark.py

```diff
@@ -0,0 +1,187 @@
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Runs YCSB against Cassandra."""
```

Comment:
That's rather short - mind adding a bit of additional info such as a link to the project page or source, and expanding the "YCSB" acronym?